src/main/java/com/dongtronic/diabot/data/mongodb/NightscoutDTO.kt

```diff
@@ -1,20 +1,21 @@
 package com.dongtronic.diabot.data.mongodb
 
 import com.dongtronic.diabot.data.ConversionDTO
-import java.time.ZonedDateTime
+import com.dongtronic.diabot.logic.nightscout.TrendArrow
+import java.time.Instant
 
 data class NightscoutDTO (
-    var glucose: ConversionDTO? = null,
-    var delta: ConversionDTO? = null,
-    var dateTime: ZonedDateTime? = null,
-    var deltaIsNegative: Boolean = false,
-    var low: Int = 0,
-    var bottom: Int = 0,
-    var top: Int = 0,
-    var high: Int = 0,
-    var trend: Int = 0,
-    var iob: Float = 0.0F,
-    var cob: Int = 0,
-    var units: String = "",
-    var title: String = "Nightscout"
+        var glucose: ConversionDTO? = null,
+        var delta: ConversionDTO? = null,
+        var dateTime: Instant? = null,
+        var deltaIsNegative: Boolean = false,
+        var low: Int = 0,
+        var bottom: Int = 0,
+        var top: Int = 0,
+        var high: Int = 0,
+        var trend: TrendArrow = TrendArrow.NONE,
+        var iob: Float = 0.0F,
+        var cob: Int = 0,
+        var units: String = "",
+        var title: String = "Nightscout"
```

Comment:
Why the double indentation?

Reply:
IntelliJ automatically reformatted it to have double indentation. I can try to adjust the project's code style to avoid this styling.

Reply:
Weird but let's not try to tell intellij what to do too much.